cli/index.ts

```diff
@@ -137,63 +158,63 @@ createYargsCli({
+          let isCompiling = false;
 
-            // Add event listeners.
-            watcher
-              .on("ready", () => console.log("Watcher ready for changes..."))
-              .on("error", error => console.error(`Watcher error: ${error}`))
-              .on("all", () => {
-                if (timeoutID || isCompiling) {
-                  // don't recompile many times if we changed a lot of files
-                  clearTimeout(timeoutID);
-                } else {
-                  console.log("Watcher recompile project...");
-                }
+          // Initialize watcher.
+          const watcher = chokidar.watch(projectDir, {
+            ignored: /node_modules/,
+            persistent: true,
+            ignoreInitial: true,
+            awaitWriteFinish: {
+              stabilityThreshold: 1000,
+              pollInterval: 200
+            }
+          });
 
-                timeoutID = setTimeout(() => {
-                  clearTimeout(timeoutID);
+          // Add event listeners.
+          watcher
+            .on("ready", () => console.log(commandOutputFmtString, "Watching for changes..."))
+            .on("error", error => console.error(`Error: ${error}`))
+            .on("all", () => {
+              if (timeoutID || isCompiling) {
+                // don't recompile many times if we changed a lot of files
+                clearTimeout(timeoutID);
+              } else {
+                console.log(commandOutputFmtString, "Recompiling project...");
+              }
 
-                  if (!isCompiling) {
-                    // recompile project
-                    isCompiling = true;
-                    compileProject(projectDir, defaultSchemaOverride, assertionSchemaOverride).then(
-                      () => {
-                        console.log("Watcher ready for changes...");
-                        isCompiling = false;
-                      }
-                    );
-                  }
-                }, RECOMPILE_DELAY);
-              });
-          }
-        });
+              timeoutID = setTimeout(() => {
+                clearTimeout(timeoutID);
+
+                if (!isCompiling) {
+                  // recompile project
+                  isCompiling = true;
+                  compileProject(projectDir, defaultSchemaOverride, assertionSchemaOverride).then(
+                    () => {
+                      console.log(commandOutputFmtString, "Watching for changes...");
+                      isCompiling = false;
+                    }
+                  );
+                }
+              }, RECOMPILE_DELAY);
+            });
+        }
       }
     },
     {
       format: "build [project-dir]",
       description:
-        "Build the dataform project. Produces JSON output describing the execution graph.",
-      positionalOptions: [projectDirOption],
+        "Build the dataform project, using the current state of the configured warehouse to compute final SQL.",
```

Comment:
Could we say: "Build the dataform project using warehouse configuration to compute final SQL" ?

Reply:
Added in PR I'm about to send to you